projects/sensory_interface/public/app.js

```diff
@@ -1,12 +1,20 @@
 // initialize Audio context on page load.
 let AudioContext = window.webkitAudioContext || window.AudioContext;
 let audioContext = new AudioContext();
+let oscillator = null;
+// This variable is not used currently.
+let source = null;
+// A variable which holds the current table cell under the touch point.
+let currentCellUnderTouchPoint = null;
+// A variable for the timeout.
```

Comment:
Comments should add extra information. In this case, the variable name has the same information, so the comment is unnecessary.